libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -68,6 +68,14 @@ const AP_Param::GroupInfo AC_Avoid::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("BEHAVE", 5, AC_Avoid, _behavior, AP_AVOID_BEHAVE_DEFAULT),
 
+    // @Param: ACTIVE_SPD
+    // @DisplayName: Active avoidance speed
+    // @Description: Active avoidance max speed (m/s)
+    // @Units: m/s
+    // @Range: 0 2
+    // @User: Standard
+    AP_GROUPINFO("ACTIVE_SPD", 6, AC_Avoid, _active_speed, 0.0f),
```

Comment:
I wonder if it might be better to call it BACKUP_SPD or BCKAWY_SPD.  Of course the vehicle is not always moving in reverse but I think it might be easier for users to guess what it is for.

Reply:
This actually doesn't do anything anymore, it currently just uses the loiter speeds I guess, might be a good idea to re-instate this tho.

Reply:
Yes, I think it would be good to set a maximum backaway speed.  I think about 50cm/s would be a good default value.  My worry is just that there is some kind of sensor error especially when the vehicle is just taking off that could lead to the vehicle flying in a random direction potentially towards the pilot.

Reply:
done, also improved name